.github/PULL_REQUEST_TEMPLATE/control_plane_template.md

```diff
@@ -17,6 +17,7 @@ What's the purpose of this PR? Check the specific option that applies. This is *
   - [ ] New API version for an existing resource provider. (If API spec is not defined in TypeSpec, the PR should have been created in adherence to [OpenAPI specs PR creation guidance](https://aka.ms/azsdkdocs/createopenapispec)).
   - [ ] Update existing version for a new feature. (This is applicable only when you are revising a private preview API version.)
   - [ ] Update existing version to fix OpenAPI spec quality issues in S360.
+  - [ ] Convert existing swagger spec to Typespec (do not combine this with implementing a new API version).
```

Comment:
```suggestion
  - [ ] Convert existing OpenAPI spec to TypeSpec spec (do not combine this with implementing a new API version).
```

We purposefully avoid using the term swagger as deprecated and incorrect. We agreed to this internally within Azure SDK plus there is this public blog post:

- https://swagger.io/blog/api-strategy/difference-between-swagger-and-openapi/

Reply:
Seems to me we should link here to some doc about TypeSpec conversions if we want to avoid unnecessary support burden.

I found this:

https://azure.github.io/typespec-azure/docs/next/howtos/migrate-swagger/get-started

Our TypeSpec page links to the page above from `Resources`, but that's it:

- [aka.ms/azsdk/typespec](https://aka.ms/azsdk/typespec)

I would not add this paragraph until we sort out the documentation, as I am usually one of the few people on first line of support fielding questions.

@mikeharder @MattGertz @rkmanda